eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -21,3 +21,196 @@ function Get-NamepspacesFromDll($dllPath) {
     }
     return $namespaces | Sort-Object -Unique
 }
+
+function DownloadNugetPackage($package, $version, $destination) {
+    # $PackageSourceOverride is a global variable provided in
+    # Update-DocsMsPackages.ps1. Its value can set a "customSource" property.
+    # If it is empty then the property is not overridden.
+    $customPackageSource = $env:PACKAGESOURCEOVERRIDE
+    $defaultDownloadUri = "https://www.nuget.org/api/v2/package/$package/$version"
+    try {
+        if ($customPackageSource){
+            $publicFeedsUrl = "https://pkgs.dev.azure.com/azure-sdk/public/_apis/packaging/feeds/azure-sdk-for-net/nuget/packages/$package/versions/$version/content?api-version=6.0-preview.1"
```

Comment:
Should we use the custom feed as the first part of this string? As it stands it seems like we are hardcoding this and it will need to be in sync with the other location we are setting. 'https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json' If we ever pass in another value there then this will be broken.

Reply:
The location we hard-coded before can't use for download. I did not find how we utilize the public feeds for downloading purpose.

We can add download-devOpsPackages in `Invoke-DevOps.ps1` so we do not hardcoded anymore. A little extra work as the changes are inside of common script.

Reply:
What I want to make sure is that we don't hard code this download path here but we actually pass in another feed in our customPackageSource property. So we need to somehow make those be in-sync.